Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,25 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about diving?"
+    m 3wub "It's very exciting. Just imagine you are diving."
+    m 3wub "You can see a wonderful underwater world full of different marine life."
```

Comment:
If you have the same expression in a row, you don't have to specify the following identical expressions.